packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -87,44 +83,27 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, any> = {
       // rectangular button defaults
       ...(!text && {
         outline: 0,
-        borderWidth: pxToRem(borderWidth),
+        borderWidth,
         borderStyle: 'solid',
         borderColor,
         boxShadow,
+
         ':hover': {
           color: colorHover,
           backgroundColor: backgroundColorHover,
           borderColor: borderColorHover,
         },
-        ...(isFromKeyboard && {
-          ':focus': {
-            color: colorFocus,
-            backgroundColor: backgroundColorFocus,
-            borderColor: borderColorFocus,
-            ':after': {
-              content: '""',
-              position: 'absolute',
-              top: `-${pxToRem(borderWidth * 2)}`,
-              right: `-${pxToRem(borderWidth * 2)}`,
-              bottom: `-${pxToRem(borderWidth * 2)}`,
-              left: `-${pxToRem(borderWidth * 2)}`,
-              borderWidth: pxToRem(borderWidth),
-              borderStyle: 'solid',
-              borderColor: borderColorFocusIndicator,
-              borderRadius: borderRadiusFocused,
-            },
-          },
-        }),
-        ...(!isFromKeyboard && {
-          ':focus': {
-            ':active': {
-              color: colorActive,
-              backgroundColor: backgroundColorActive,
-              borderColor: borderColorActive,
-              boxShadow: 'none',
-            },
-          },
-        }),
+
+        ':focus': {
+          boxShadow: 'none',
+          ...(isFromKeyboard
+            ? {
+                color: colorFocus,
+                backgroundColor: backgroundColorFocus,
+                ...borderFocusStyles,
+              }
+            : { ':active': { backgroundColor: backgroundColorActive } }),
```

Comment:
Why we decided to still use the background active color, but not the foreground (the colorActive variable). When we press the button now, we may have some regressions..

Reply:
we don't foreground colors were the same as regular (not focused) view, no point in having so many variables that actually bring nothing